docs/conf.py

```diff
@@ -28,10 +28,25 @@
 # it
 
 # Copy the docs from Cliquet inside these ones.
-destination = os.path.join(__HERE__, 'api', 'cliquet')
+# XXX how do we copy on 2.x ?
+destination = os.path.join(__HERE__, 'api', '1.x', 'cliquet')
```

Comment:
That's the problem with having the HTTP API documentation inside a Python package. The HTTP API documentation should be separated from the implementation. I understand that it's easier to maintain if it's in the Kinto repo, but the kinto package releases should not interfere with the HTTP API versions.

The target readership does not care about the implementation. They want the API doc, and they want to be able to see how 1.1 is different from 2.3 for instance.